Sources/Brave/Frontend/Browser/User Scripts/UserScriptType.swift

```diff
@@ -47,6 +47,9 @@ enum UserScriptType: Hashable {
     let order: Int
     /// We need to pass this setting over to the javascript code
     let isDeAMPEnabled: Bool
+    /// This enables some debugging on scriplets
+    /// - Note: For now this is disabled but we can add some debug tools to enable this in the future
+    let isScripletDebugEnabled = false
```

Comment:
IIRC, as of recently uBO is not planning to use this anymore. I just haven't gotten to removing it from brave-core yet.

I'd leave it out for now and just use an empty constructor above, i.e. `new Map()`.

Reply:
ill do that in that case